ClangSharp/ClangSharp.csproj

```diff
@@ -5,4 +5,8 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />

```

Comment:
Going to remove this. I had experimented with `MarshalAs(UnmanagedType.ByValArray`, but that doesn't work with return types without additional fixups.